Sources/FaunaDB/HighPrecisionTime.swift

```diff
@@ -34,10 +45,17 @@ public struct HighPrecisionTime {
 
 extension HighPrecisionTime {
 
+    /// Converts a `Foundation.Date` into an instance of `HighPrecisionTime`.
+    /// - Note: Ignores anything lower than seconds precision.
```

Comment:
Similar to above, except in the other direction. This time, the behavior is a bit more clear: "Sub-second components default to 0." or something?